erts/doc/src/erl.xml

```diff
@@ -382,6 +382,28 @@
           similar to <c><![CDATA[code:add_pathsz/1]]></c>. See
           <seealso marker="kernel:code">code(3)</seealso>.</p>
       </item>
+      <tag><c><![CDATA[-proto_dist Proto]]></c></tag>
+      <item>
+        <p>Specify a protocol for Erlang distribution.</p>
+	    <taglist>
+	      <tag><c>inet_tcp</c></tag>
+              <item>
+                  <p>TCP over IPv4 (the default)</p>
+              </item>
+	      <tag><c>inet_ssl</c></tag>
+              <item>
+                  <p>distribution over SSL</p>
+              </item>
```

Comment:
I just noticed this: the module was renamed from `inet_ssl` to `inet_tls` in R15. Also the description should probably say "distribution over TLS/SSL".